src/components/transactions/Withdraw/WithdrawActions.tsx

```diff
@@ -27,31 +25,19 @@ export const WithdrawActions = ({
   blocked,
   sx,
 }: WithdrawActionsProps) => {
-  const { lendingPool } = useTxBuilderContext();
-  const { currentChainId: chainId, currentMarketData } = useProtocolDataContext();
   const { currentAccount } = useWeb3Context();
```

Comment:
or here

Reply:
Moved current account from Actions to zustand,